akka-docs-dev/rst/java/migration-guide-1.0-2.x-java.rst

```diff
@@ -422,7 +422,7 @@ should be replaced by:
 .. includecode:: code/docs/MigrationsJava.java#query-param
 
 SynchronousFileSource and SynchronousFileSink
-============================================
+=============================================
```

Comment:
I? Sure I do; so I always fix those. But never realised it's a real rendering problem that I'm fixing ;)